kaggle-classification/hparam_config.yaml

```diff
@@ -1,5 +1,6 @@
 trainingInput:
   ## BASIC_GPU uses single NVIDIA Tesla K80 GPU.
+  pythonVersion: '3.5'
```

Comment:
Is this python version required for hyperparameter tuning?

Reply:
No, but I think we want our python versions to be consistent. I can remove this from this pr though